package.json

```diff
@@ -1,11 +1,10 @@
 {
-  "name": "root",
+  "name": "cli-packages",
   "private": true,
   "scripts": {
     "bootstrap": "lerna bootstrap",
     "build": "lerna run build",
     "ci-docs": "yarn sf-ci-docs",
-    "ci-test": "nyc yarn test",
```

Comment:
Not sure why that was here so I removed it.  sfdx-dev-packages didn't have this so making the repos consistent.